src/layman/layer/filesystem/input_chunk.py

```diff
@@ -138,10 +140,10 @@ def get_layer_info(workspace, layername):
     if info:
         files_to_upload = info['files_to_upload']
         file_names = [file['input_file'] for file in files_to_upload]
-        if len(file_names) == 1 and input_file.get_compressed_main_file_extension(file_names[0]):
+        if len(file_names) >= 1 and any(input_file.get_compressed_main_file_extension(file_name) for file_name in file_names):
```

Comment:
if `any(...)` is truthy, `len(file_names) >= 1` is always truthy too. So `len(file_names) >= 1` is unnecessary.